docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md

```diff
@@ -0,0 +1,69 @@
+---
+title:  Hue community 2023
+author: Hue Team
+type: post
+date: 2023-03-02T00:00:00+00:00
+url: /blog/hue-community-2023/
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - community
+  - development
+
+---
+
+Hello everyone!
+
+Last year was a bit slow for the open source community around Hue but we are planning to change that going forward in 2023. There will be more frequent releases, improved issue handling process and a wider initiative to engage more people outside of Cloudera. We want the Hue community to be vibrant, useful and inclusive and we will try to lower the threshold for those who want to engage by simplifying, encouraging and rewarding contributions in all forms. 
+
+We start off these improvements by establishing a process for how to handle incoming issues for our [Github repository](https://github.com/cloudera/hue/issues). Although Hue is open source and used in the Hadoop services of all the major cloud providers, it is still mainly developed and maintained by the Hue core team at Cloudera. Upstream code changes and releases for the open source community coexist with more frequent downstream releases like [Cloudera Data Platform (CDP)](https://www.cloudera.com/products/cloudera-data-platform.html) and we always try to balance these two sides of Hue. When it comes to handling incoming issues like bug reports, requests for enhancements or questions, our goal is to increase the amount that gets resolved by the help of external committers and experts, but we also don’t want issues to go stale in lack of community engagement. Therefore we have put in place a process to make sure that:
+
+  * Incoming issues are filed using improved templates to speed up resolution times.
+  * The core team looks at all incoming issues and labels them as either “bug” or “configuration”. 
+    Incoming issues that are in fact questions get transformed to Github discussions.
```

Comment:
* Incoming issues of the type "questions" get transformed into GitHub discussions.